lms/static/js/student_account/views/RegisterView.js

```diff
@@ -277,6 +278,9 @@
                     $('#register-confirm_email').bind('cut copy paste', function(e) {
                         e.preventDefault();
                     });
+                    $('#register-confirm_password').bind('cut copy paste', function(e) {
```

Comment:
Can we prevent "confirm password" field from pasting to enforce manual typing?

Reply:
@tuanh118 it is exactly what this line does